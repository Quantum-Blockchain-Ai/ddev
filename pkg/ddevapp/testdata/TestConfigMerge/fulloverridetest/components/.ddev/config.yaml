database:
  type: mariadb
  version: 10.5
php_version: 5.6
webserver_type: apache-fpm
use_dns_when_possible: true
additional_hostnames: [baseone, basetwo, basethree]
