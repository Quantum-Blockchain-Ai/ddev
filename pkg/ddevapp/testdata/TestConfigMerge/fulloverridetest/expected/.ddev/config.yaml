database:
  type: mariadb
  version: 10.5
php_version: 5.6
webserver_type: apache-fpm
use_dns_when_possible: false
additional_hostnames: [overrideone, overridetwo, z-one]
